utils/helpers.js

```diff
@@ -0,0 +1,4 @@
+export const addDays = (date, days) => {
```

Comment:
moved this over from the component library because the default value wasn't getting set in the webstore local state.